client/helpers.go

```diff
@@ -195,6 +195,25 @@ func ParseSigs(sigs []byte, hash []byte, validators []common.Address) ([]uint8,
 	return vs, rs, ss, valIndexes, nil
 }
 
+// Concatenate arrays of (v, r, s) back to one signature
+func ConcatSigs(vs []uint8, rs [][32]byte, ss [][32]byte) []byte {
```

Comment:
Move this to `loomchain/builtin/plugins/gateway/gateway.go`

Reply:
do you mean **transfer-gateway**/builtin/plugins/gateway/gateway.go?